backstop_data/engine_scripts/puppeteer/clickAndHoverHelper.js

```diff
@@ -20,4 +20,7 @@ module.exports = async (page, scenario) => {
   if (postInteractionWait) {
     await page.waitFor(postInteractionWait);
   }
+
+  // make sure we're not hovering over anything
+  await page.mouse.move(0, 0);
```

Comment:
It looks like this JS can be used in case _we do_ want to hover over something too, Should we put this change inside the `if (clickSelector)` ?

Reply:
Oh yeah good point haha